src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java

```diff
@@ -336,7 +337,7 @@ private void drainHolder(SessionKey key, SessionBatchHolder holder, boolean forc
             batch = holder.reset();
         }
         if (batch.isEmpty()) {
-            logger.debug("Holder for stream: '{}', direction: '{}' has been concurrently reset. Skip storing by scheduler",
+            logger.debug("Holder for session alias: '{}', direction: '{}' has been concurrently reset. Skip storing by scheduler",
```

Comment:
Must we add the `book name` property to the SessionKey class?

Reply:
This is a common idea and doesn't related to code line

Reply:
done